### ISSUE TITLE
provide js assets in the source tarball

### ISSUE THREAD

**Issue by landryb** [facette/facette#334]
just discussed this with @falzm, for 'distro' packagers itwould be super nice to provide js assets in the source tarball we grab from github so that we dont have npm/bower reaching out to internet during build - otherwise we need to repackage ourself the source tarball, manually adding the assets that were locally fetched.

**Comment by vbatoufflet** [comment]
I'm fine with this idea but I'll to take a look at it to see how I can automate this as the current source tarballs available are automatically generated by Github.

**Comment by landryb** [comment]
Yeah, and our build infra is able to fetch a tarball from the github tag, but that doesnt have the assets - i fully understand if you dont want to commit the assets to the repository, so maybe it's only a matter of providing a 'tarball' target in the Makefile that builds the assets and creates the source tarball. - after that, either the distributors use this target to create and host the tarball themselves, or upstream provides it, but that's more work for you :/

**Comment by vbatoufflet** [comment]
It's ok for me having such `tarball` target or similar as, indeed, I don't want to commit the assets in the repository 😄 

I'll try to add this on the master branch soon.
